### PR TITLE
Add and update Repo Documentation

### LINKED ISSUE #109: README in root of Repo could Use Some Updates
The README in the root of our repo could use some updates so to remove some of the links that do not need the entire *URL* showing.  
Some of these would include:  
- Linking the *CS 305* projects so that when you click on the name of the person who worked on the project, you are redirected to their project from *CS 305*.  
    - This would also include linking *DASSL* to their version of the product as well.
- Making it so that all of the member's names listed at the end of the document are links to their **GitHub** profile.  
 - Update the section `File Structure` so that it reflects some of the changes that will be made with the System Testing Documentation.
- Add in all of the badges at the top of the README
    - Like *PostgreSQL*, *Node JS*, *ZenHub*, etc...

### LINKED ISSUE #110: [IDEA] Add a README to System Testing Directory
**Is your feature request related to a problem? Please describe.**
It is not related to a problem

**Describe the solution you'd like**
Add a README to the directory `/tests/SystemTesting` to describe what is happening within the directory and all subdirectories that are created in there.

**Describe alternatives you've considered**
N/A

**Additional context**
The README would include information about what would be in each directory and what the conventions would be with all of the directories names and what would be included with each directory created. For Example:
- A directory `Gradebook v0.5.5` was created 
    - The naming convention of each of the subdirectories of `SystemTesting` would be the name of the release that it is for.
- Within the directory `Gradebook v0.5.5` would be a *copy* of the last edited version of the file `GradebookSystemTesting.md` and a file called `GradebookSystemTesting.pdf`
    - The Markdown file would contain all of the edits up to that release being made and would be needed in the directory in the event that a change is made that requires that version of Gradebook to need updates to the System Testing.
    - The PDF would be a polished *printable* version of this document so that someone who is wanting to install Gradebook that would be in a user-friendly format for reading.

### LINKED ISSUE #111: [IDEA] iREADMEs in DB and Webapp Need Updating
**Is your feature request related to a problem? Please describe.**
Not connected to any problems

**Describe the solution you'd like**
- In the directory `/src/db`:
    - Update the SQL scripts that need to be run to get Gradebook up and running so that someone installing Gradebook will run all scripts needed to get the System up and running without error.
- In the directory `/src/webapp`
    - In the `Overview` section add in all of the ways that the Server has been tested and working on.
    - Stating all of the Web Broswers and Devices that Gradebook is compatible with
        - Also stating that *Internet Explorer on Windows 10* is **not** compatible.

**Describe alternatives you've considered**
N/A

**Additional context**

### PR DESCRIPTION
**Pull request by kozlowskik**
Updates were made to various READMEs within the repo to improve the content of the documents and make it easier to click on links.  
Also, added a new README in the `SystemTesting` directory to explain what is/will be going on in the directory.
***
This PR will close the following issues once the new features are pulled to Dev:
- Fixes #109 
- Fixes #110 
- Fixes #111

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +36,10 @@ The src directory will be split into two sub-directories:
 ---  
 
 ## Members
-Bruno Desilva  
+[Bruno Desilva](https://github.com/DasilvaB)
```
Comment (griffine) [inline README.md]: Bruno's last name is misspelled.

File: src/db/README.md [hunk 2]
```diff
@@ -9,6 +9,9 @@ https://creativecommons.org/licenses/by-nc-sa/4.0/
 
 PROVIDED AS IS. NO WARRANTIES EXPRESSED OR IMPLIED. USE AT YOUR OWN RISK.
 
+Modified by **Team GEEKS** (*CS298-01 Spring 2019 @ WCSU*)  
+Bruno DaSilva, Elly Griffen, Chris Fitzgerald and Kenneth Kozlowski
```
Comment (griffine) [inline src/db/README.md]: Cris' first name and my last name are misspelled.

File: src/webapp/README.md [hunk 3]
```diff
@@ -7,21 +7,21 @@ Data Science & Systems Lab (DASSL), Western Connecticut State University (WCSU)
 Licensed to others under CC 4.0 BY-SA-NC:   
 https://creativecommons.org/licenses/by-nc-sa/4.0/
 
-PROVIDED AS IS. NO WARRANTIES EXPRESSED OR IMPLIED. USE AT YOUR OWN RISK.
+PROVIDED AS IS. NO WARRANTIES EXPRESSED OR IMPLIED. USE AT YOUR OWN RISK.  
 
+Modified by **Team GEEKS** (*CS298-01 Spring 2019 @ WCSU*)  
+Bruno DaSilva, Elly Griffen, Chris Fitzgerald and Kenneth Kozlowski
```
Comment (griffine) [inline src/webapp/README.md]: Cris' first name and my last name are misspelled.

File: README.md (hunk 1 above)
Comment (kozlowskik) [reply README.md]: Corrected

File: src/db/README.md (hunk 2 above)
Comment (kozlowskik) [reply src/db/README.md]: Corrected

File: src/webapp/README.md (hunk 3 above)
Comment (kozlowskik) [reply src/webapp/README.md]: Corrected

**Comment by griffine** [comment]
Other than the spelling errors that were fixed, I think it looks good. I do think that we should be consistent with which names we use (Ken vs Kenneth and Elly vs Eliot) to avoid any possible confusion to others. I have no preference for which name is used for me.

**Review by fitz-cris — DISMISSED** [review]

**Review by griffine — APPROVED** [review]

**Review by DasilvaB — APPROVED** [review]

**Pull request merged**